cluster/src/main/scala/io/snappydata/cluster/ExecutorInitiator.scala

```diff
@@ -83,6 +84,11 @@ object ExecutorInitiator extends Logging {
       lock.notify()
     }
 
+    // Test hook. Not to be used in other situations
+    def waitForExecutor(retry: Boolean = true): Unit = testLock.synchronized {
```

Comment:
might be better to make this private[cluster] and have the test in the same package

Reply:
It is a bit problematic, as the test uses package level private methods from memory package.